playground/pages/without.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+const show = ref(false)
+const CreditCardError = ref()
+const CreditCardToken = ref()
+</script>
+
+<template>
+    <h1>Load Mollie script lazily</h1>
+    <div>On this page the scripts load lazily, if the button is clicked.</div>
+    <div>To test this, the option <i>isIncludedGlobally: false</i> has to be set in nuxt config.</div>
+
+    <p>
+        <button @click="show = !show">{{show ? 'Hide' : 'Show'}} payment components</button>
+        <p v-if="show">
+            <LazyShopwareFrontendsCreditCard
+                submit-button-label="Save"
+                locale="en_US"
+                :submit-disabled="!!CreditCardToken"
+                @submit="
+                    (token) => {
+                        CreditCardToken = `${token} ✔️`
+                        CreditCardError = null
+                    }
+                "
+                @error="
+                    (message) => {
+                        CreditCardError = `${message} ❌`
+                    }
+                "
+            />
+        </p>
+    </p>
```

Comment:
The linter complains here `Parsing error: x-invalid-end-tag`

Reply:
fixed :)